gensim/models/keyedvectors.py

```diff
@@ -1385,6 +1385,35 @@ def init_sims(self, replace=False):
             logger.info("precomputing L2-norms of word weight vectors")
             self.vectors_norm = _l2_norm(self.vectors, replace=replace)
 
+    def relative_cosine_similarity(self, wa, wb, topn=10):
+        """Compute the relative cosine similarity between two words given top-n similar words,
+        by Artuur ... <https://ufal.mff.cuni.cz/pbml/105/art-leeuwenberg-et-al.pdf>.
```

Comment:
Use full reference please (not this "...").

Reply:
Done.